client/components/tinymce/plugins/simple-payments/dialog/index.jsx

```diff
@@ -282,10 +282,9 @@ class SimplePaymentsDialog extends Component {
 
 		const { siteId, dispatch, translate } = this.props;
 
-		wpcom
-			.site( siteId )
-			.post( paymentId )
-			.delete()
+		const productAPI = wpcom.site( siteId ).post( paymentId );
+		productAPI.delete()
```

Comment:
I don't think we are using "WP REST API" for this. `wpcom.js` is using the standard WP.com API which I'm not sure supports the `force` parameter. Doesn't look like that from the docs: https://developer.wordpress.com/docs/api/1.1/post/sites/%24site/posts/%24post_ID/delete/